crates/host/src/wasmbus/mod.rs

```diff
@@ -181,20 +188,24 @@ impl Queue {
         host_key: &KeyPair,
     ) -> anyhow::Result<Self> {
         let host_id = host_key.public_key();
-        let (registries, pings, links, queries, auction, commands, inventory) = try_join!(
+        let (registries, pings, links, queries, auction, commands, inventory, config) = try_join!(
             nats.subscribe(format!("{topic_prefix}.{lattice_prefix}.registries.put",)),
             nats.subscribe(format!("{topic_prefix}.{lattice_prefix}.ping.hosts",)),
             nats.queue_subscribe(
                 format!("{topic_prefix}.{lattice_prefix}.linkdefs.*"),
                 format!("{topic_prefix}.{lattice_prefix}.linkdefs",)
             ),
             nats.queue_subscribe(
-                format!("{topic_prefix}.{lattice_prefix}.get.*"),
+                format!("{topic_prefix}.{lattice_prefix}.get.>"),
```

Comment:
This seems out-of-scope 🤔 why is this change made?

Reply:
It isn't out of scope actually! If you look at the handler for getting config values, it matches multiple parts of the remaining subject so it wasn't actually listening for requests